guide/english/documentation/index.md

```diff
@@ -5,10 +5,45 @@ title: Code documentation
 
 Code documentation is a way developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
 
+## Why Documentation:
+
+## 1.For You: 
+    ~You will be using your code in 6 months
```

Comment:
```suggestion
- You will be using your code in 6 months
```
Please change all the lines like this to the format I have given above, would make it into a list